src/components/ThemeButton.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { forwardRef } from "react";
+import { Button as NextUIButton, Link } from "@nextui-org/react";
+
+type NextUIButtonProps = React.ComponentProps<typeof NextUIButton>;
+
+type ButtonProps = Omit<NextUIButtonProps, "color"> & {
+  label?: string | React.ReactNode;
+  color?: "primary" | "secondary" | "tertiary";
+  isSelected?: boolean;
+};
+
+type Ref = HTMLButtonElement;
+
+// override NEXT-UI default styles
+const basedStyles = `flex items-center p-2 gap-1 h-[40px] align-middle`;
+const removedTransitionStyles = `transition-none hover:transition-none data-[pressed=true]:transition-none data-[pressed=true]:transition-none data-[pressed=true]:scale-100 data-[pressed=true]:skew-x-0 data-[pressed=true]skew-y-0 data-[pressed=true]:rotate-0 data-[pressed=true]:translate-x-0 data-[pressed=true]:translate-y-0`;
+
+const btnColorStyles = {
+  primary: `bg-green-600 hover:bg-green-700 active:bg-green-800 text-white`,
+  primaryDisabled: `bg-green-600/50 text-white`,
```

Comment:
This is just a nit, but did you explore the possibly of utilizing tailwind's builtin `disabled` classes, [found here](https://tailwindcss.com/docs/hover-focus-and-other-states#disabled)? For example, this may be simplified to: 
`primary: bg-green-600 hover:bg-green-700 active:bg-green-800 text-white disabled:bg-green-600/50 disabled:text-white`, which would also simplify the `getButtonColorStyles ` logic from 31:65. In any case, great work!

Reply:
Thanks, Paul! According to the [accessibility issue](https://github.com/CodeForPhilly/vacant-lots-proj/issues/288#issuecomment-1968079059), we don't actually make the button `disabled`. So, I'm not sure if the `disabled` class would be enabled for this case.